de.dlr.sc.overtarget.language.ui/src/de/dlr/sc/overtarget/language/ui/handler/ResolveTargetHandler.java

```diff
@@ -25,13 +26,29 @@
 import org.eclipse.ui.handlers.HandlerUtil;
 import org.eclipse.ui.part.FileEditorInput;
 import org.eclipse.ui.texteditor.ITextEditor;
+import org.eclipse.xtext.builder.EclipseOutputConfigurationProvider;
 
-import de.dlr.sc.overtarget.language.ui.util.TargetPlatformHelper;
+import com.google.inject.Inject;
+import com.google.inject.Provider;
+
+import de.dlr.sc.overtarget.language.ui.internal.LanguageActivator;
+import de.dlr.sc.overtarget.language.util.TargetPlatformHelper;
 
 public class ResolveTargetHandler extends AbstractHandler implements IHandler {
```

Comment:
Since you renamed the Button... Guess it makes sense to rename this class as well? TargetActivationHelper?

Reply:
I called it TargetActivationHandler, since it is a handler not a helper, ok?